pkg/apis/sources/v1alpha1/mongodbsource_validation_test.go

```diff
@@ -26,18 +26,18 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-func TestCouchDbSourceValidation(t *testing.T) {
+func TestMongoDbSourceValidation(t *testing.T) {
```

Comment:
add UTs for the secret as well.
You should be able to see examples on knative-gcp sources

Reply:
Question about the data type of secret/ what it will end up being

Reply:
TODO: unit test